packages/teleport/src/Console/DocumentNodes/useNodes.ts

```diff
@@ -156,10 +152,26 @@ export default function useNodes({ clusterId, id }: stores.DocumentNodes) {
   }
 
   function getNodeSshLogins(serverId: string) {
-    return results.logins.map(login => ({
-      login,
-      url: consoleCtx.getSshDocumentUrl({ serverId, login, clusterId }),
-    }));
+    const node = results.nodes.find(node => node.id == serverId);
+    return makeOptions(clusterId, node);
+  }
+
+  function makeOptions(clusterId: string, node: Node | undefined) {
+    const nodeLogins = node?.sshLogins || [];
+    nodeLogins.unshift('root');
```

Comment:
I'm not sure that we can always assume that there will always be a `root` login?

Reply:
You're right, thanks for pointing that out. I missed that `root` is added to the top of the list if it is present in the logins in the first place. 

Fixed in https://github.com/gravitational/webapps/pull/873/commits/076772774718c896a6aa3704c6132885cfadaf6a